alpha/scripts/utils/permissions/addPermissionsAndItems.php

```diff
@@ -247,7 +247,9 @@ function addParameterPermissionItem($itemCfg)
 			}
 					
 			$setterCallback = array ( $item ,"set{$key}");	
-			call_user_func_array( $setterCallback , array ($value ) );
+			if (method_exists($item,'set'.$key)){
```

Comment:
Done. Thanks,

May the source be with you,

Jess Portnoy

Kaltura Connect Virtual Summits 2015 - now on VOD! [connect.mediaspace.kaltura.com]

On Tue, 24 Nov 2015, Tan-Tan wrote:

> 
> In alpha/scripts/utils/permissions/addPermissionsAndItems.php:
> 
> > @@ -247,7 +247,9 @@ function addParameterPermissionItem($itemCfg)
> >  			}
> > 
> >  			$setterCallback = array ( $item ,"set{$key}"); 
> > -			call_user_func_array( $setterCallback , array ($value ) );
> > +			if (method_exists($item,'set'.$key)){
> 
> Please add an else that logs the error.
> 
> —
> Reply to this email directly or view it on GitHub.[ACH7Xit3UPKwboIlRJBn2PpJXWXNtX8Iks5pJBgOgaJpZM4GnuM-.gif]
> 
> 
>